packages/test-harness/src/launchVscodeAndRunTests.ts

```diff
@@ -81,10 +81,7 @@ export async function launchVscodeAndRunTests(extensionTestsPath: string) {
       // don't bother.  Can be re-enabled if we ever need it; on windows it only
       // hangs some of the time, so might be enough to get a crash dump when you
       // need it.
-      launchArgs:
-        useLegacyVscode || os.platform() === "win32"
-          ? undefined
-          : [`--crash-reporter-directory=${crashDir}`, `--logsPath=${logsDir}`],
```

Comment:
Why not leave these? Could be useful for debugging

Reply:
Oh I'm just testing right now. Shouldn't make it a draft.